src/docs/product/performance/queries.mdx

```diff
@@ -65,9 +74,18 @@ To view more details, click on a query from the table to open its **Query Summar
 
 The gif below provides a detailed walkthrough of the **Queries** page.
 
-<div style="position: relative; padding-bottom: calc(48.68055555555556% + 41px); height: 0; width: 100%"><iframe src="https://demo.arcade.software/v66mxydrZRsNxRgNMYUl?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;" title="Performance - Queries - Queries Page Walkthrough"></iframe></div>
-
-</br>
+<div style="position: relative; padding-bottom: calc(48.68055555555556% + 41px); height: 0; width: 100%">
+  <iframe
+    src="https://demo.arcade.software/v66mxydrZRsNxRgNMYUl?embed"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"
+    title="Performance - Queries - Queries Page Walkthrough"
+  ></iframe>
+</div>
```

Comment:
I removed it when I converted to MDX because it made the entire page fail to build, not now it seems fine, bringing it back! Does Arcade allow customizing how much padding there is under the arcade embed?